primerpedia.css

```diff
@@ -11,6 +11,24 @@ body, h2, .error {
 	text-align: center;
 }
 
+#notification {
+	position: fixed;
+	top: 0;
+	left: 0;
+
+	width: 100%;
```

Comment:
For this to work properly, we also need to add `box-sizing: border-box;`, otherwise the banner becomes too long since the width, by default, is taken as defining the content box (i.e. without the padding).